.template/addons/crud/app/assets/stylesheets/layouts/_application.scss

```diff
@@ -0,0 +1,92 @@
+body {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.app {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 1;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    justify-content: space-between;
+  }
+
+  &__main {
+    padding: rem(16px);
+    margin: rem(16px);
+  }
+
+  &__footer {
+    grid-area: footer;
+    padding: rem(8px);
+  }
+
+  &__nav_bar_top {
+    grid-area: navBar;
+    border-bottom: solid var(--bs-border-width) var(--bs-gray-400);
+  }
+
+  &__side_bar {
+    grid-area: sideBar;
+    display: flex;
+    flex-flow: column;
+    padding: rem(32px);
+    background-color: var(--bs-light);
+    width: rem(280px);
+    height: 100%;
+
+    .dropdown-toggle {
+      outline: 0;
+    }
+
+    .btn-toggle {
+      padding: 0.25rem 0.5rem;
+      font-weight: 600;
+      color: var(--bs-emphasis-color);
+      background-color: transparent;
+    }
+    .btn-toggle:hover,
+    .btn-toggle:focus {
+      color: rgba(var(--bs-emphasis-color-rgb), 0.85);
+      background-color: var(--bs-tertiary-bg);
+    }
+
+    .btn-toggle::before {
```

Comment:
IMO, I think we can put all the states like `:hover`, `:focus`, or `::before` pseudo-element, ... inside the 
`.btn-toggle` selector.

Reply:
3270c1d :white_check_mark: